apps/web/components/booking/pages/AvailabilityPage.tsx

```diff
@@ -563,7 +568,10 @@ const AvailabilityPage = ({ profile, eventType }: Props) => {
                       <div>
                         <InformationCircleIcon className="mr-[10px] ml-[2px] -mt-1 inline-block h-4 w-4 text-gray-400" />
                       </div>
-                      <p>{eventType.description}</p>
+                      <div
+                        className="markdown-body"
+                        dangerouslySetInnerHTML={{ __html: md.render(eventType?.description) }}
+                      />
```

Comment:
As mentioned above, can we handle this on the server? Also does this sanitizes inputs? Does this prevents XSS attacks?  We should very cautious on using `dangerouslySetInnerHTML` for user entered data.